monitor/linuxmonitor/processor.go

```diff
@@ -36,6 +41,8 @@ func NewCustomLinuxProcessor(storePath string, collector collector.EventCollecto
 		netcls:            cgnetcls.NewCgroupNetController(releasePath),
 		contextStore:      contextstore.NewContextStore(storePath),
 		storePath:         storePath,
+		regStart:          regexp.MustCompile("^[a-zA-Z0-9_].{0,11}$"),
```

Comment:
Can you comment the intent? nobody likes regex :)

Reply:
Because its more compact and elegant :)